src/fidesops/api/v1/endpoints/connection_endpoints.py

```diff
@@ -102,6 +108,13 @@ def get_connections(
                 ConnectionConfig.description.ilike(f"%{escape_like(search)}%"),
             )
         )
+
+    if connection_type:
+        query = query.filter(ConnectionConfig.connection_type.in_(connection_type))
+
+    if disabled is not None:
+        query = query.filter(ConnectionConfig.disabled == disabled)
+
```

Comment:
It looks like there are 2 more filters that are needed for the frontend. 

One is `Testing Status`. This would be filtering based on the `last_test_succeeded` field. I think it needs to be able to filter based on `true`, `false`, and `null`. `null` would be for connections that have never been tested.

The second is `System Type`. In the mockups it looks it's for filtering for all SAAS connections or all Database connections. I'm not sure if this is possible with the current data model. I think it would conflict with the `connection_type` filter you added because it would be a second filter based on the same field. Does that seem correct?

Reply:
Thanks @TheAndrewJackson - I can add both testing status and system type. it's fine if it's a second filter on the same field, just important to note that if the user selects "saas" and then filters for "postgres" then they'll get no results.

Reply:
Re: Testing status, let's make sure we're being careful about the "null" status.  Usually a None would mean, "don't filter", and in this case, you're wanting it to mean "we never tested it".  Maybe we do string filters here, "passed", "failed", "untested", and None, or similar.

Reply:
OK I added filtering on `?test_status=passed`, `?test_status=failed`, and`?test_status=untested`, and then there's
`?system_type=saas` and `?system_type=database`.  

If someone filters on conflicting fields, they'll get no results: `?system_type=saas&connection_type=postgres`.